libraries/dagster-contrib-chroma/README.md

```diff
@@ -0,0 +1,106 @@
+# dagster-contrib-chroma
+
+A dagster module that provides integration with [chroma](https://www.trychroma.com/).
+
+This module provides the `ChromaLocalResource` (for a self-hosted instance), 
+and the `ChromaHttpResource` (to connect to a Chroma server via HTTP).
```

Comment:
this looks like it needs to be updated - since the local vs http now happens at the config layer